homeassistant/components/broadlink/remote.py

```diff
@@ -92,7 +95,7 @@
         return
     hass.data[DOMAIN][COMPONENT].append(unique_id)
 
-    api = broadlink.rm((host, DEFAULT_PORT), mac_addr, None)
+    api = broadlink.rm((host, DEFAULT_PORT), mac_addr, devtype)
```

Comment:
Does it make a difference to pass on `0x272A` instead of `None`?

Reply:
We should not pass None in this non-optional parameter. Things are working now, but there is no guarantee that this will not lead to undefined behavior in the future.